README.md

```diff
@@ -30,14 +30,30 @@ git clone https://github.com/mongodb/mongo-tools
 cd mongo-tools
 ```
 
-To use build/test scripts in the repo, you *MUST* set GOROOT to your Go root directory.
+To use the build/test scripts in this repository, you **_must_** set GOROOT to your Go root directory. This may depend on how you installed Go.
 
 ```
 export GOROOT=/usr/local/go
 ```
 
-Run `./make build` to build all the tools, placing them in the `bin`
-directory.
+Run `./make build` to build all the tools, placing them in the `bin` directory.
+
+You can also build a subset of the tools using the `-tools` option. For example, `./make build -tools=mongodump,mongorestore` builds only `mongodump` and `mongorestore`.
+
+Updating Dependencies
+---------------
+Starting with version 100.3.0, the tools use `go mod` to manage dependencies. All dependencies are listed in the `go.mod` file and are directly vendored in the `vendor` directory.
+
+In order to make changes to dependencies, you first need to change the `go.mod` file. You can manually edit that file to add/update/remove entries, or you can run the following in the repository directory:
+
+```
+go mod edit -require=<package>@<version>  # for adding or updating a dependency
+go mod edit -droprequire=<package>        # for removing a dependency
+```
+
+Then run `go mod vendor -v` to reconstruct the `vendor` directory to match the changed `go.mod` file.
+
+Optionally, run `go mod tidy -v` to ensure that the `go.mod` file matches the vendored code.
```

Comment:
Is that really what `tidy` does? I thought `go mod tidy` removed unused dependencies. Saying it ensures the `go.mod` file matches the vendored code seems misleading.

Reply:
Ah, I pulled the phrasing from [the Go docs entry for go mod](https://golang.org/ref/mod#go-mod-tidy) (first sentence) but I misinterpreted it. Thanks for catching this.

It does remove unused dependencies, but it could also add new ones. Would phrasing it like this seem alright? Feel free to suggest something else too.
> Optionally, run `go mod tidy -v` to ensure that the `go.mod` file matches the `mongo-tools` source code.

Reply:
Yes that is better phrasing.